boards/arm/mec2016evb_assy6797/doc/mec2016evb_assy6797.rst

```diff
@@ -0,0 +1,212 @@
++=======+======+======+======+======+======+=======+=======+
+| 1-2   | 1-2  | 1-2  | 1-2  | 1-2  | 1-2  | 1-2   | 1-2   |
++-------+------+------+------+------+------+-------+-------+
+
++-------+-------+------+------+------+-------+
+| JP56  | JP57  | JP58 | JP60 | JP61 | JP102 |
++=======+=======+======+======+======+=======+
+| 1-2   | 1-2   | 1-2  | 1-2  | 1-2  | 2-3   |
++-------+-------+------+------+------+-------+
+
+This jumpers configure VCC Powergood, nRESETI and JTAG_STRAP respectively.
+
++-------+-------+------+
+| JP1   | JP2   | JP51 |
++=======+=======+======+
+| 1-2   | 1-2   | 2-3  |
++-------+-------+------+
+
+Each column of the following table illustrates how to enable UART0, JTAG,
+PVT SPI, SHD SPI and LED0-3 respectively.
+
++-------+-------+---------+--------+--------+
+| JP27  | JP10  | JP34    | JP75   | JP68   |
++=======+=======+=========+========+========+
+| 11-12 | 2-3   | 2-3     | 2-3    | 1-2    |
++-------+-------+---------+--------+--------+
+| 8-9   | 5-6   | 5-6     | 5-6    | 3-4    |
++-------+-------+---------+--------+--------+
+|       | 8-9   | 8-9     | 8-9    | 5-6    |
++-------+-------+---------+--------+--------+
+|       | 11-12 | 11-12   | 11-12  | 7-8    |
++-------+-------+---------+--------+--------+
+|       |       | 14-15   | 14-15  |        |
++-------+-------+---------+--------+--------+
+|       |       | 17-18   | 17-18  |        |
++-------------------------------------------+
+
+Jump settings for MEC170x 144WFBGA Socket DC Assy 6801 Rev B1p0
+===============================================================
+
+The jumper configuration explained above covers the base board. Now the cpu
+board requires the following settings.
+
++-------+-------+
+| JP1   | JP2   |
++=======+=======+
+| 1-2   | 2-3   |
++-------+-------+
+
+Programming and Debugging
+*************************
+
+This board comes with a Cortex ETM port which facilitates tracing and debugging
+using a single physical conection.  In attidion, it comes with sockets for
+Jtag only sessions.
+
+Flashing
+========
+
+#. Connect the SPI Dongle ASSY 6791 to J36 (SPI dongle) in order to flash and
+   boot from SHD SPI NOR. Then proceed to flash using Dediprog SF100 or a
+   similar tool for flashing SPI chips. Remember that SPI MISO/MOSI are
+   swapped on dediprog headers!
+
+#. Run your favorite terminal program to listen for output. Under Linux the
+   terminal should be :code:`/dev/ttyACM0`. For example:
+
+   .. code-block:: console
+
+      $ minicom -D /dev/ttyACM0 -o
+
+   The -o option tells minicom not to send the modem initialization
+   string. Connection should be configured as follows:
+
+   - Speed: 115200
+   - Data: 8 bits
+   - Parity: None
+   - Stop bits: 1
+
+#. Connect the MEC2016EVB_ASSY_6797 board to your host computer using the
+   UART0 port. Then build :ref: `hello_world` application. It is important
+   to generate a binary with a new load address, for example do the following:
```

Comment:
The easy way to get a code-block is to use a double colon at the end of the sentence:
```suggestion
   to generate a binary with a new load address, for example do the following::
```

Reply:
done